x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_table.tsx

```diff
@@ -134,6 +136,35 @@ const CustomGridBody = memo(
   }
 );
 
+const ViewError = ({ error }: { error: Error }) => (
+  <>
+    <EuiFlexGroup
+      gutterSize="s"
+      alignItems="center"
+      css={css`
+        height: 1lh;
+      `}
+    >
+      <EuiIcon type="error" color="danger" />
+      <EuiText
+        color="subdued"
+        size="xs"
+        css={css`
+          line-height: unset;
+          font-weight: ${euiThemeVars.euiFontWeightSemiBold};
```

Comment:
Absolutely agree with limiting custom styling, here though we're doing it to obtain an error callout that exactly matches the line height of the table cell, hiding the error details in the overflow area so that it only shows when opening the cell popover (↙↗ hover cell action) 🙂

Collapsed cells only shows the first row:
<img width="226" alt="image" src="https://github.com/elastic/kibana/assets/18363145/85c47c2b-41f5-4687-8261-8d9731c17cd8">

Expanded cells show the full error:
<img width="443" alt="image" src="https://github.com/elastic/kibana/assets/18363145/4dec9df3-18e9-4fb8-9d0e-2639d345856c">

The custom font weight is there to try and match what they're doing in the `controls`  embeddable where they use a similar UI to handle errors locally, but using a `<strong>` tag is absolutely fine as well 🙂

Forgot the `FlexItem`, thanks!

Reply:
Thanks for explaining! Maybe it'd be good to add a comment here so it's clear when people look back this code